pkg/build/build.go

```diff
@@ -497,6 +497,10 @@ func (b *Build) IsBuildLess() bool {
 // ConfigFileExternalRef calculates ExternalRef for the melange config
 // file itself.
 func (b *Build) ConfigFileExternalRef() (*purl.PackageURL, error) {
+	// TODO(luhring): This is now the second implementation of finding the commit
```

Comment:
Are these TODOs worthy of an issue? If I were looking at the second implementation, would I know of this one?

Reply:
>Are these TODOs worthy of an issue?

Yes! I'll open one and link it here...

>If I were looking at the second implementation, would I know of this one?

Good point — do you have a specific solution in mind? I can add a similar comment to the other implementation, for example.

Reply:
c7f9e86 adds a cross-referencing comment

Reply:
https://github.com/chainguard-dev/melange/issues/1460